app/views/charge.njk

```diff
@@ -173,14 +173,12 @@
                     {% if allowedCards %}
                       {% for currentAllowedCard in allowedCards %}
                         <li
-                      class="{{ currentAllowedCard.brand }}"
-                      data-key="{{ currentAllowedCard.brand }}"
-                      data-credit="{{ currentAllowedCard.credit }}"
-                      data-debit="{{ currentAllowedCard.debit }}"
-                  >
-                          <span class="govuk-warning-text__assistive">
-                            {{ currentAllowedCard.brand }}
-                          </span>
+                          class="{{ currentAllowedCard.brand }}"
+                          data-key="{{ currentAllowedCard.brand }}"
+                          data-credit="{{ currentAllowedCard.credit }}"
+                          data-debit="{{ currentAllowedCard.debit }}"
+                        >
+                          <img src="/images/transparent-accessibiliity.gif" alt="{{ currentAllowedCard.brand | replace("-", " ") | title }}" />
```

Comment:
Is there not a `.label` property on card brands that will do this for you? 

If it's driven directly by the [Connector resource](https://github.com/alphagov/pay-connector/blob/master/src/main/java/uk/gov/pay/connector/cardtype/model/domain/CardBrandLabelEntity.java) then it should.

Reply:
I didn't know that - will take a look, thanks

Reply:
I will merge this in and then look at that.
Thanks

Reply:
Does not seem to be a `.label` property.
Only:
-Brand
- Debit
- Credit